crates/ruma-events/src/kinds.rs

```diff
@@ -453,19 +453,16 @@ pub struct StrippedStateEvent<C: PossiblyRedactedStateEventContent> {
     /// the same tuple replaces the previous one.
     pub state_key: C::StateKey,
 
-    /// The globally unique identifier for the event.
-    ///
-    /// This field is usually stripped, but some events might include it.
-    #[cfg(feature = "unstable-msc4319")]
-    #[ruma_event(default)]
-    pub event_id: Option<OwnedEventId>,
-
     /// Timestamp on the originating homeserver when this event was sent.
     ///
     /// This field is usually stripped, but some events might include it.
     #[cfg(feature = "unstable-msc4319")]
     #[ruma_event(default)]
     pub origin_server_ts: Option<MilliSecondsSinceUnixEpoch>,
+
+    /// Additional key-value pairs not signed by the homeserver.
+    #[cfg(feature = "unstable-msc4319")]
+    pub unsigned: Option<Raw<crate::StateUnsigned<C>>>,
```

Comment:
This is a bit ugly, ideally we should be able to just use `StateUnsigned<C>`, or better something like `StaticStateEventContent::Unsigned`, but those would require breaking changes.

Reply:
It's not using the `Option` that changes the bounds, it is using `Raw`. `Raw` doesn't need any bounds on the inner type to be deserializable in itself. It only needs bounds on the inner type, when calling `.deserialize()` on it.

We use an `Option` because `Raw` doesn't implement `Default`.

And sure I could add it but serde_json treats `Option` as having the `default` attribute by default.